assets/scss/_header.scss

```diff
@@ -0,0 +1,57 @@
+.header {
+    background-image: url('../images/header-bg-mobile.svg');
+    background-size: 100% auto;
+    background-repeat: no-repeat;
+
+    @include media-queries('tablet') {
+        background-image: url('../images/header-bg.svg');
+    }
+}
+
+.nav__list {
+    min-height: 20vw;
+    display: flex;
+    align-items: center;
+    list-style: none;
+    padding: 0 5vw 5.5vw;
```

Comment:
Padding will look little different on different devices. It's the best to set some constant value (like `rem`). It will only change if user has bigger font size then